app/feature/odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/di/OdysseyModule.kt

```diff
@@ -57,4 +59,7 @@ val odysseyModule = module {
   viewModel<SingleItemPayoutViewModel> { (singleItemPayout: ClaimFlowDestination.SingleItemPayout) ->
     SingleItemPayoutViewModel(singleItemPayout, get<ClaimFlowRepository>())
   }
+  viewModel<SelectContractViewModel> { (selectedContract: String?, contractOptions: List<LocalContractContractOption>) ->
```

Comment:
This could get the entire `(ClaimFlowDestination.SelectContract)` instead of `(String, List<LocalContractContractOption>)`. Should help for if the options change in the future, and also helps with issues like if you want to send over 2 things of type `String` since koin has no way to know which one is which otherwise.

We've done the same above for SingleItemViewModel for example.